optimum_benchmark/trackers/energy.py

```diff
@@ -78,6 +59,47 @@ def __truediv__(self, scalar: float) -> "Energy":
             total=self.total / scalar,
         )
 
+    @staticmethod
+    def aggregate(energies: List["Energy"]) -> "Energy":
```

Comment:
```suggestion
    def aggregate(energies: List[Energy]) -> Energy:
```

It could just be Energy, no? Energy is defined above so no need for a forward reference

Reply:
I don't think you can use a class to annotate its own methods, let me check

Reply:
not possible